ansys/dpf/core/field_base.py

```diff
@@ -294,18 +306,18 @@ def get_entity_data_by_id(self, id):
         """
         index = self.scoping.index(id)
         if index < 0:
-            raise ValueError(f'The id {id} must be greater than 0')
+            raise ValueError(f'The ID {id} must be greater than 0.')
         return self.get_entity_data(index)
 
     def append(self, data, scopingid):
-        """add an entity data to the existing data
+        """Add an entity data to the existing data.
 
         Parameters
         ----------
         data : list of int, double or array
```

Comment:
```suggestion
        data : list of int, double, or array
```
I'm assuming we're using the oxford comma (I've seen you add that in the past).